From 64c9aff005c7f7c68d62d23ef5b526871c8976b9 Mon Sep 17 00:00:00 2001
From: Bin Han <bin.b.han@intel.com>
Date: Tue, 21 Oct 2014 10:34:01 +0800
Subject: [PATCH 05/10] css2400b0_v21: [WORKAROUND]do not hang system when
 refcount check fails

BZ: 226252

ICG BZ 4276 is used to tracking this issue, will apply this workaround before
a propoer solution.

Change-Id: I53a2c376bac34894aa5da3a40a1224180a416d43
Tracked-On: https://jira01.devtools.intel.com/browse/IMINAN-8527
Signed-off-by: Bin Han <bin.b.han@intel.com>
Reviewed-on: https://android.intel.com:443/290968
---
 .../css2400b0_v21/base/refcount/src/refcount.c     |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21/base/refcount/src/refcount.c b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21/base/refcount/src/refcount.c
index 9fe9b48..eaf44e7 100644
--- a/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21/base/refcount/src/refcount.c
+++ b/drivers/external_drivers/camera/drivers/media/pci/atomisp2/css2400b0_v21/base/refcount/src/refcount.c
@@ -202,8 +202,8 @@ bool ia_css_refcount_decrement(int32_t id, hrt_vaddress ptr)
 		IA_CSS_ERROR("id %x, ptr 0x%x entry %p entry->id %x entry->count %d\n",
 			id, ptr, entry, entry->id, entry->count);
 	else
-		IA_CSS_ERROR("entry NULL\n");
-	assert(false);
+		IA_CSS_ERROR("entry NULL for ptr 0x%x\n", ptr);
+	/* assert(false); */
 
 	return false;
 }
-- 
1.7.9.5

